pkgs/applications/blockchains/zcash/default.nix

```diff
@@ -20,13 +41,21 @@ rustPlatform.buildRustPackage.override { inherit stdenv; } rec {
       --replace "linker = \"aarch64-linux-gnu-gcc\"" ""
   '';
 
-  cargoHash = "sha256-6uhtOaBsgMw59Dy6yivZYUEWDsYfpInA7VmJrqxDS/4=";
+  cargoHash = "sha256-u9bulmUeoQibNJIB/DP3wosaPVUJeDs9K4DlR+zu3Ug=";
 
-  nativeBuildInputs = [ autoreconfHook cargo cxx-rs hexdump makeWrapper pkg-config ];
+  nativeBuildInputs = [
+    autoreconfHook
+    cargo
+    cxx-rs
+    git
```

Comment:
Thanks for this fix!

The stable version fails to build for me with:
```
checking for git... git_notfound
configure: error: "Required program git was not found"
```

Reply:
To be clear, you're saying that `git` is not found even though I added it to `nativeBuildInputs`?